dts/bindings/power-domain/nxp,scu-pd.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP SCU-managed power domain
+
+compatible: "nxp,scu-pd"
+
+include: power-domain.yaml
+
+properties:
+  nxp,rsrc-id:
+    type: int
+    required: true
+    description:
+      Number used by the firmware running on the SCU to identify
```

Comment:
firmware magic numbers not allowed in DT, very bad practice, could change with firmware update